settings/base.py

```diff
@@ -101,7 +101,7 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
-    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    #'django.middleware.clickjacking.XFrameOptionsMiddleware',
```

Comment:
@waseem-omar can you give me info on why this is commented out? I'm sure its a good reason, but would love to understand.